api/api.go

```diff
@@ -129,6 +130,13 @@ func (h *Handler) getRate(c *gin.Context) {
 		}
 	}
 
+	msgGasLimitParam := c.DefaultQuery("msgGasLimit", "0")
+	msgGasLimit, err := util.MsgGasLimitChecker(msgGasLimitParam)
```

Comment:
Not really sure what this function does by the name.
If it only converts from string to *big.Int then it doesn't really have anything to do with gasLimit and it is only FormatBigInt()

Reply:
It converts string to *big.Int and then check if msgGasLimit is less than 0